master/internal/rm/kubernetesrm/pods.go

```diff
@@ -129,6 +129,11 @@ type reattachPodResponse struct {
 	started     *sproto.ResourcesStarted
 }
 
+// messages that are sent by the informer.
+type podStatusUpdate struct {
```

Comment:
moved this over to `pods.go `from `informers.go` because it's something that will eventually be refactored, but now the step at which it interacts with the informers is in the `pods.go` file within the `startPodInformer` function.